ci/etc/minio-no-create-bucket-policy.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Hmm, this file doesn't seem used anywhere. Did you forget to add some other files?

Reply:
It was intended for an eventual regression test (which I've now added).